src/UI/Component/Popover/Popover.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace ILIAS\UI\Component\Popover;
+
+use ILIAS\UI\Component\Component;
+use ILIAS\UI\Component\Signal;
+use ILIAS\UI\Component\Triggerable;
+use ILIAS\UI\Implementation\Component\Popover\ReplaceContentSignal;
+
+/**
+ * Describes the Popover component
+ */
+interface Popover extends Component, Triggerable {
+
+	/**
+	 * Get the title of the popover.
+	 *
+	 * @return string
+	 */
+	public function getTitle();
+
+	/**
+	 * Get the components representing the content of the popover.
+	 *
+	 * @return Component[]
+	 */
+	public function getContent();
+
+	/**
+	 * Get the position of the popover.
+	 *
+	 * @return string
```

Comment:
Ok. But why expose a `getPosition` on the interface then, if I only get strings that I cannot reliably understand as a consumer. So it's either expose `getPosition` and according constants or not expose both of them.